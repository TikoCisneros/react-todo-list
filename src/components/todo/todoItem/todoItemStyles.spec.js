import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import {
  ItemContainer,
  ItemCheck,
  ItemName,
  ItemDelete,
} from './todoItemStyles';

describe('TodoItem components', () => {
  it('ItemContainer render successfully', () => {
    const tree = renderer.create(<ItemContainer />).toJSON()
    expect(tree).toMatchSnapshot()
  });
  it('ItemCheck render successfully', () => {
    const tree = renderer.create(<ItemCheck />).toJSON()
    expect(tree).toMatchSnapshot()
  });
  it('ItemName completed render successfully', () => {
    const tree = renderer.create(<ItemName completed/>).toJSON()
    expect(tree).toMatchSnapshot()
  });
  it('ItemName render successfully', () => {
    const tree = renderer.create(<ItemName />).toJSON()
    expect(tree).toMatchSnapshot()
  });
  it('ItemDelete render successfully', () => {
    const tree = renderer.create(<ItemDelete />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
