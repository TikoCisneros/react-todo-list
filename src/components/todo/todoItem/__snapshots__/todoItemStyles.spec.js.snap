// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoItem components ItemCheck render successfully 1`] = `
.c0 {
  margin-right: 10px;
  height: 18px;
  width: 18px;
  cursor: pointer;
}

.c0:focus {
  outline: 0;
}

<input
  className="c0"
/>
`;

exports[`TodoItem components ItemContainer render successfully 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 10px 20px;
  width: 80%;
  border-bottom: 1px solid #cecece;
  font-weight: 100;
  font-size: 18px;
}

.c0:hover .erase {
  display: inline-block;
}

<div
  className="c0"
/>
`;

exports[`TodoItem components ItemDelete render successfully 1`] = `
.c0 {
  background-color: #fb4f42;
  border: none;
  border-radius: 2px;
  color: white;
  padding: 5px 10px;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: none;
  font-size: 12px;
  cursor: pointer;
}

.c0:hover {
  background-color: #fb4f42c9;
}

.c0:focus {
  outline: 0;
}

<button
  className="c0"
/>
`;

exports[`TodoItem components ItemName completed render successfully 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  color: #cdcdcd;
  font-style: italic;
  -webkit-text-decoration: line-through;
  text-decoration: line-through;
}

<p
  className="c0"
/>
`;

exports[`TodoItem components ItemName render successfully 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  color: #333333;
  font-style: normal;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<p
  className="c0"
/>
`;
